src/main/scala/viper/server/frontends/lsp/CommandProtocol.scala

```diff
@@ -0,0 +1,50 @@
+/**
+  * This Source Code Form is subject to the terms of the Mozilla Public
+  * License, v. 2.0. If a copy of the MPL was not distributed with this
+  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  *
+  * Copyright (c) 2011-2020 ETH Zurich.
+  */
+
+package viper.server.frontends.lsp
+
+object C2S_Commands {
```

Comment:
The protocol described in this file must be in sync with the one written in Typescript for the Viper IDE client, right?
If so, please add a comment in this file that explains that this is the case, and refers to the corresponding protocol file in the viper-ide repo.

Reply:
The same applies to ```S2C_Commands```, obviously.